src/marshmallow/fields.py

```diff
@@ -1479,20 +1500,36 @@ def __init__(self, precision: str = SECONDS, **kwargs):
             )
             raise ValueError(msg)
 
+        serde_types = (
+            int,
+            float,
+        )
+
+        if serde_type not in serde_types:
+            msg = f"The (de)serialization must be one of {serde_types}"
+            raise ValueError(msg)
```

Comment:
I don't see a test for this, but I'm not sure it is required.

Reply:
I put one into `test_serialization.py` although maybe its not the correct place for it...

Reply:
Oh, right. Didn't see it.